subsys/bluetooth/controller/ll_sw/ull_master.c

```diff
@@ -434,13 +434,21 @@ void ull_master_setup(memq_link_t *link, struct node_rx_hdr *rx,
 	pdu = (void *)((struct node_rx_pdu *)rx)->pdu;
 	chan_sel = pdu->chan_sel;
 
+#if defined(CONFIG_BT_CTLR_PRIVACY)
+	u8_t init_addr[BDADDR_SIZE];
```

Comment:
General comment, what is the convention when it comes to variable declaration, should it be at the beginning of the locality of reference or can it be as-is close to its first use?

Reply:
It should be at the beginning of the function if possible, otherwise wherever it makes sense with #fidef statements